include/platform/messagepool.h

```diff
@@ -69,7 +69,7 @@ extern "C" {
  * @param[in] aBufferSize          The size in bytes of a Buffer object.
  *
  */
-void otPlatMessagePoolInit(int aMinNumFreeBuffers, size_t aBufferSize);
+void otPlatMessagePoolInit(uint16_t aMinNumFreeBuffers, size_t aBufferSize);
```

Comment:
Oh, sorry, I completely missed the notification for this.  I'll make that change now.